src/plugins/gpgme/README.md

```diff
@@ -0,0 +1,71 @@
+- infos = Information about gpgme plugin is in keys below
+- infos/author = Peter Nirschl <peter.nirschl@gmail.com>
+- infos/licence = BSD
+- infos/provides = crypto
+- infos/needs =
+- infos/recommends =
+- infos/placements = postgetstorage presetstorage
+- infos/status = unittest configurable unfinished experimental memleak
+- infos/metadata = crypto/encrypt gpg/binary
+- infos/description = Cryptographic operations wit GnuPG Made Easy (GPGME)
+
+## Introduction
+
+The `gpgme` plugin is a filter plugin that enables users to encrypt values before they are
+persisted and to decrypt values after they have been read from a backend.
+The encryption and decryption is designed to work transparently.
+
+The cryptographic operations are performed by GnuPG via the `libgpgme` library.
+
+## Dependencies
+
+- `libgpgme11` version 1.10 or later
+
+## Build Information
+
+The plugin has been tested on Ubuntu 18.04 with `libgpgme` version 1.10.
+
+## Examples
+
+You can mount the plugin like this:
+
+	kdb mount test.ecf /t gpgme "encrypt/key=DDEBEF9EE2DC931701338212DAF635B17F230E8D"
+
+Now you can specify a key `user/t/a` and protect its content by using:
+
+    kdb set user/t/a
+    kdb setmeta user/t/a crypt/encrypt 1
+    kdb set user/t/a "secret"
+
+The value of `user/t/a` (for this example: "secret") will be stored encrypted.
+You can still access the original value by using `kdb get`:
+
+    kdb get user/t/a
+
+
+## Configuration
+
+### GnuPG Keys
+
+The GPG recipient keys can be specified in two ways:
+
+1. The GPG recipient key can be specified as `encrypt/key` directly.
+2. If you want to specify multiple keys, you can enumerate them under `encrypt/key`.
+
+The following example illustrates how multiple GPG recipient keys can be specified:
+
+    encrypt/key/#0
+    encrypt/key/#1
+
+
+### Textmode
+
+`gpgme` operates in textmode per default. In textmode the output of GPG is ASCII armored.
+
+Textmode can be disabled by setting `/gpgme/textmode` to `0` in the plugin configuration.
```

Comment:
Seems like we need to add /gpgme/textmode to dump. Or we add a generic /crypto/textmode which is supported by both fcrypt and gpgme?

Reply:
Why do we have to add it to dump? Is text mode per default a bad idea?

Reply:
See #1543 there was some bug when using fcrypt+dump together which was fixed with turning off textmode.

Did you try to use gpgme+dump, did it work?

Reply:
> Did you try to use gpgme+dump, did it work?

I did and it works just fine. At least using `gpgme` has _some_ advantage. :see_no_evil: